app/controllers/application_controller.rb

```diff
@@ -7,11 +9,19 @@ class ApplicationController < ActionController::Base
 
   def set_locale
     lang = :en
-    unless params[:locale].nil?
-      lang = params[:locale]
-    end
-    lang = :"pt-BR" if lang.downcase == "pt"
-    lang = :"pt-BR" if lang.downcase == "br"
-    I18n.locale = I18n.available_locales.include?(lang.to_sym) ? lang : I18n.default_locale
+    lang = params[:locale] unless params[:locale].nil?
```

Comment:
ApplicationController#set_locale performs a nil-check

[Read more about it here.](https://github.com/troessner/reek/blob/master/docs/Nil-Check.md)